smarts/core/mission_planner.py

```diff
@@ -299,7 +310,7 @@ def uturn_waypoints(self, pose: Pose):
             # agent at the start edge
             p0 = pose.position[:2]
             distance = (
-                10 * abs(abs(target_heading - heading) - math.pi / 2) / (math.pi / 2)
+                15 * abs(abs(target_heading - heading) - math.pi / 2) / (math.pi / 2)
```

Comment:
This curve value perhaps deserves to be part of the task attributes.

Reply:
This one shouldn't be exposed, because it's leaking implementation details rather than a configurable param to users.

Reply:
OK, if I am reading this correctly, this equation models an intent for a larger offset based on the difference between vehicle headings. Perhaps you are right here.